sysl2/sysl/codegen.go

```diff
@@ -231,12 +231,27 @@ func loadAndGetDefaultApp(root, model string) (*sysl.Module, string) {
 
 // GenerateCode transform input sysl model to code in the target language described by
 // grammar and a sysl transform
-func GenerateCode(rootModel, model, rootTransform, transform, grammar, start string) []*CodeGenOutput {
+func GenerateCode(codegenParams *CmdContextParamCodegen) []*CodeGenOutput {
 	var codeOutput []*CodeGenOutput
 
-	mod, modelAppName := loadAndGetDefaultApp(rootModel, model)
-	tx, transformAppName := loadAndGetDefaultApp(rootTransform, transform)
-	g := readGrammar(grammar, "gen", start)
+	logrus.Warnf("root-model: %s\n", *codegenParams.rootModel)
```

Comment:
Is there a specific reason for using Warning logs? I think Info would be better.

Reply:
ok